DataFormats/CSCDigi/interface/CSCALCTDigi.h

```diff
@@ -95,6 +105,11 @@ class CSCALCTDigi {
   /// set wiregroup number
   void setWireGroup(unsigned int wiregroup) { keywire_ = wiregroup; }
 
+  /// Distinguish Run-1/2 from Run-3
+  bool isRun3() const { return version_ == Version::Run3; }
+
+  void setRun3(bool isRun3) { isRun3 ? version_ = Version::Run3 : Version::Legacy; }
```

Comment:
this is wrong; it is the equivalent of
```c++
if (isRun3) {
  (void) version_ = Version::Run3;
} else {
  (void) Version::Legacy;
}
```
while what you want is probably
```c++
if (isRun3) {
  version_ = Version::Run3;
} else {
  version_ = Version::Legacy;
}
```
or, on one line
```c++
version_ = isRun3 ? Version::Run3 : Version::Legacy;
```

So
```suggestion
  void setRun3(bool isRun3) { version_ = isRun3 ? Version::Run3 : Version::Legacy; }
```

Reply:
Thanks. I'll fix it.